### PR TITLE
Swift 4 support with RxSwift/RxBlocking 4.0

### PR DESCRIPTION
**Pull request by tomisacat**
Nothing but change dependencies updated to RxSwift/RxBlocking 4.0

### CODE REVIEW

**Comment by dvlprliu** [comment]
I think you should also update `osx_image: ` in `.travis.yml` file to `xcode9` or `xcode9.1` to pass CI

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-osx_image: xcode8.3
+osx_image: xcode9
 language: objective-c
 podfile: Demo/Podfile
 cache: cocoapods
 before_install:
-  - gem install cocoapods -v '1.2.1'
+  - gem install cocoapods -v '1.3.1'
   - pod repo update
 script: cd Demo ; set -o pipefail && xcodebuild -workspace 'Demo.xcworkspace' -scheme 'Demo' -configuration 'Debug' -sdk iphonesimulator -destination platform='iOS Simulator',OS='10.3',name='iPhone 6s' build test  | xcpretty -c --test
```
Comment (oronbz) [inline .travis.yml:8]: Hi @tomisacat, can you just update `OS='10.3',name='iPhone 6s'` to iOS11 and iPhone 8 to make sure travis pass?

**Review by oronbz — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (oronbz) [reply .travis.yml:8]: I've already fixed it on https://github.com/RxSwiftCommunity/RxNimble/pull/27, since it's pretty urgent. Thanks a lot for this man!

**Comment by oronbz** [comment]
I've already fixed it

File: .travis.yml (hunk 1 above)
Comment (tomisacat) [reply .travis.yml:8]: Sorry about just noticing your comment, glad it helps!

**Pull request closed without merging**